sdks/java/io/google-cloud-platform/src/main/java/org/apache/beam/sdk/io/gcp/datastore/DatastoreV1.java

```diff
@@ -1152,7 +1161,14 @@ public void startBundle(StartBundleContext c) {
 
     @ProcessElement
     public void processElement(ProcessContext c) throws Exception {
```

Comment:
Just curious if this impacts `deletes` in any way, since this `DoFn` is used across writes and deletes.